cluster/gce/gci/configure-helper.sh

```diff
@@ -826,10 +828,12 @@ function start-kube-apiserver {
   fi
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
 
-  if [[ -n "${KUBE_USER:-}" ]]; then
+  if [[ -n "${KUBE_USER:-}" || ! -e /etc/srv/kubernetes/abac-authz-policy.jsonl ]]; then
     local -r abac_policy_json="${src_dir}/abac-authz-policy.jsonl"
     remove-salt-config-comments "${abac_policy_json}"
-    sed -i -e "s/{{kube_user}}/${KUBE_USER}/g" "${abac_policy_json}"
+    if [[ -n "${KUBE_USER:-}" ]]; then
+      sed -i -e "s/{{kube_user}}/${KUBE_USER}/g" "${abac_policy_json}"
```

Comment:
You end up with a meaningless `"user":"{{kube_user}}"` line. I guess I could make it a little nicer and just knock out the whole line when `KUBE_USER` isn't set:
````
else
  sed -i -e "s/.*{{kube_user}}.*//g" "${abac_policy_json}"
````